docs/configuration.md

```diff
@@ -470,6 +512,8 @@ Apart from these, the following properties are also available, and may be useful
     application ID and {{EXECUTOR_ID}} will be replaced by executor ID. For example, to enable
     verbose gc logging to a file named for the executor ID of the app in /tmp, pass a 'value' of:
     <code>-verbose:gc -Xloggc:/tmp/{{APP_ID}}-{{EXECUTOR_ID}}.gc</code>
+
+    <code>spark.executor.defaultJavaOptions</code> prepended (without any parameter override) to this configuration.
```

Comment:
... will be prepended ...

what do you mean by "without any parameter override"?

Reply:
> ... will be prepended ...

Fixed.

> what do you mean by "without any parameter override"?

`defaultJavaOptions`="key=value1"
`extraJavaOptions`="key=value2"

Result will be `key=value1 key=value2`

Wanted to make it clear the result won't be `key=value2`.
If you think it can be expressed in a better way suggest it.

Reply:
I think maybe just leaving the part about parameter overriding is more clear, as that is what prepending means.  Its just not clear what parameters and what overriding you're talking about, which ends up making it more confusing.

If you do want to explain it in more detail, I think you've got to spend more time going into those details, with a small example (as you did in this comment here).  But IMO not necessary

Reply:
Well, you're right prepend is quite explicit. Will remove it...